terraform/global/s3/main.tf

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+resource "aws_s3_bucket" "terraform_state" {
+  bucket = "app-state-${account_id}"
```

Comment:
Sorry I should be more explicit here but instead it should look like this

```
data "aws_caller_identity" "current" {}

resource "aws_s3_bucket" "terraform_state" {
  bucket = "subhub-state-${data.aws_caller_identity.current.account_id}"
}
```

Something like that